tests/Unit/Core/Form/IdentifiableObject/DataProvider/ProductFormDataProviderTest.php

```diff
@@ -1226,15 +1247,11 @@ private function getDefaultOutputData(): array
             'description' => [
                 'description' => [],
                 'description_short' => [],
+                'manufacturer' => NoManufacturerId::NO_MANUFACTURER_ID,
```

Comment:
This is weird, manufacturer and categories should be in the same order in the default data and the data for product, and you correctly fixed this in the `ProductFormDataProvider` which is a good thing, we should keep consistency in the array fields' order (that matches the form type's order as much as possible)

Besides the tests are written to detect if the expected order is not respected, and yet there seems to be a difference between the implemented code and the expected result here